README.md

```diff
@@ -17,32 +17,19 @@ Installation
 Sponsor
 -------
 
-Symfony 6.3 is [backed][27] by
+Symfony 6.4 is [backed][27] by
 - [SensioLabs][28]
-- [Shopware][29]
-- [Les-Tilleuls.coop][30]
-- [basecom][31]
+- [packagist.com][29]
 
 As the creator of Symfony, **SensioLabs** supports companies using Symfony,
 with an offering encompassing consultancy, expertise, services, training, and
 technical assistance to ensure the success of web application development projects.
 
-**Shopware** offers you cutting-edge, highly adaptable ecommerce solutions trusted
-by the world's most acclaimed brands. Create outstanding customer experiences,
-innovate fast, and accelerate your growth in the ever-evolving space of digital
-commerce. You decide how far you want to go, and we'll be by your side.
+Private **Packagist.com** is a fast, reliable, and secure Composer repository for
+your private packages. It mirrors all your open-source dependencies for better
+availability and monitors them for security vulnerabilities.
 
-**Les-Tilleuls.coop** is a team of 70+ Symfony experts who can help you design,
-develop and fix your projects. We provide a wide range of professional services
-including development, consulting, coaching, training and audits. We also are
-highly skilled in JS, Go and DevOps. We are a worker cooperative!
-
-As a professional software service provider, **basecom** implements customized
-solutions in the areas of e-commerce, PIM solutions and web portals. With their
-experience and certified expertise, they have been one of the most renowned
-Symfony specialists in Germany for many years.
-
-Help Symfony by [sponsoring][32] its development!
+Help Symfony by [sponsoring][30] its development!
```

Comment:
I would suggest putting that link before the sponsors themselves in the list of links, so that its index does not need to be updated each time the number of sponsors changes (all other links that don't change between versions are also before them even if they appear after the sponsor section, providing the same benefit)